wp-content/themes/core/components/blocks/gallery_grid/css/dialog.pcss

```diff
@@ -0,0 +1,268 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Component: Dialog
+ *
+ * ----------------------------------------------------------------------------- */
+
+.c-dialog {
```

Comment:
Hmm.... this component's css is quite a collection of other component's styles. I think you need to more clearly define what is specific to a generic `dialog` and move the rest of the styles to other component's styles or in the case of the swiper styles, move those to that integration's styles.